olp-cpp-sdk-core/src/thread/ThreadPoolTaskScheduler.cpp

```diff
@@ -59,9 +62,41 @@ void SetCurrentThreadName(const std::string& thread_name) {
 #endif  // OLP_SDK_HAVE_PTHREAD_SETNAME_NP
 }
 
+struct PrioritizedTask {
+  TaskScheduler::CallFuncType function;
+  uint32_t priority;
+};
+
+struct ComparePrioritizedTask {
+  bool operator()(const PrioritizedTask& lhs,
+                  const PrioritizedTask& rhs) const {
+    return lhs.priority < rhs.priority;
+  }  // namespace
```

Comment:
```suggestion
  }
```

Reply:
Oops, not even sure how that happened, sorry. Fixed! :-)